### PR TITLE
Add FranceConnect as FS 

### PR DESCRIPTION
**Pull request by Anna-Livia**
This PR:
- Replaces france_connect view, url and form with the official France connect journey
- Separate views in different files
- Add integration test for the FranceConnect journey

This PR also clarifies the environment variables. They have been changed on click CI and will need changing on the server.

### CODE REVIEW

File: aidants_connect_web/views/FC_as_FS.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        "gender",
+        "birthplace",
+        "birthcountry",
+    ]
+
+    parameters = (
+        f"response_type=code"
+        f"&client_id={fc_id}"
+        f"&redirect_uri={fc_callback_uri}"
+        f"&scope={'openid' + ''.join(['%20' + scope for scope in fc_scopes])}"
```
Comment (thimy) [inline aidants_connect_web/views/FC_as_FS.py]: Why do you add an empty string here?

Comment (Anna-Livia) [reply aidants_connect_web/views/FC_as_FS.py]: Are you referencing `''.join`? 
- if yes : the expression:
```py
string = ''.join(['%20' + scope for scope in fc_scopes]
```
is equivalent to 
```py
string = ''
for scope in fc_scopes:
string = '' + '%20' + scope
```
the empty string is the separator as we don't want spaces in our url.
https://docs.python.org/2/library/string.html#string.join

however, if might work to write 
```suggestion
        f"&scope={'openid' + ''%20''.join([scope for scope in fc_scopes])}"
```

- If not: Could you tell me which empty string

Comment (thimy) [reply aidants_connect_web/views/FC_as_FS.py]: Yes that was the string, now I understand that it's used to initialize the string, thanks :)

Comment (Anna-Livia) [reply aidants_connect_web/views/FC_as_FS.py]: I made a mistake in the first version of my comment. I corrected it.
The empty string is not only initializing the string, it is also use to separate the different element of the list once it is in string form.

For example : 
```py
"".join([a, b, c])
# abc

", "..join([a, b, c])
# a, b, c

"%20".join([a, b, c])
# a%20b%20c
```

**Pull request merged**